Dockerfile

```diff
@@ -1,3 +1,18 @@
+FROM gradle:7-jdk11-alpine AS build
+
+COPY --chown=gradle:gradle . /build
+WORKDIR /build
+
+ARG VERSION_TAG
+ENV versionTag=$VERSION_TAG
+RUN gradle build --no-daemon
```

Comment:
To get a fat JAR with all the dependencies bundled you'll need to use `gradle shadowJar` here instead of `gradle build`.